docs/users/developing.md

```diff
@@ -414,4 +414,4 @@ The CI matrix is currently as follows:
 +-----------------------+------------+-----------+-------------+------------+-----------+-------------+
 | Python 3.12 and newer | Pending official releases and upstream support                              | 
 +-----------------------+------------+-----------+-------------+------------+-----------+-------------+
```

Comment:
You are welcome to update this table so that 3.12 looks like 3.11 and this line applies to 3.13+, but that's not needed to get this merged